gacl/docs/manual.html

```diff
@@ -554,7 +554,7 @@ <H2 LANG="en-GB" CLASS="western"><FONT FACE="Bitstream Vera Sans Mono, sans-seri
 Falcon Passengers		Group<BR>&#9500;&#9472;Crew					Group<BR>&#9474;
 &#9500;&#9472;Han					ARO<BR>&#9474;
```

Comment:
After talking with Brady & others, this file will need some major formatting fixes.